source/backend/routes/users.js

```diff
@@ -86,4 +86,27 @@ router.delete('/:savedRecipeId', async (req, res) => {
   }
 });
 
+/* POST /user/completedRecipes */
+router.post('/completedRecipes', async (req, res) => {
+  try {
+    const { userId } = req.userInfo;
+    const { recipeId } = req.recipeInfo;
+    if (parseInt(req.params.id, 10) !== userId) {
+      return res.status(401).json({ message: 'Forbidden, acceess denied' });
+    }
+
+    const updateRecipe = req.body; 
+    updateRecipe.recipeId = recipeId;
+    await completedRecipesModel.addNewRecipe(updateRecipe, userId);
```

Comment:
What is `updateRecipe`? What is the body we are giving in the request? Is it an entire recipe? or just the recipeId? `addNewRecipe()` takes in a recipeId and a `userId`, so should you be passing `updateRecipe.recipeId` or could you just pass the `recipeId`? 

From my understanding, to add to the completed recipe table we just need the userId and the recipeId right? We can check if the user is logged in which is done on line 92. Then we just need to add that recipeId and userId to the completedRecipe table. I think this is really similar to the savedRecipes table, so i'd suggest looking at that for guidance and potentially reaching out to @joecai6 as I believe he worked on saved recipes. 

So far things look pretty good, just get those changed and I think we are on the right track

Reply:
^ To add to the completed recipe table we only need userId and recipeId. And I think theres no need for updateRecipe here since it is the same as the req.body.recipeId.

Reply:
You're absolutely right, I don't need the updateRecipe since I can get the same info from the req.body.recipeId. Thanks for pointing that out guys